### ISSUE TITLE
TipTacTalents - Lua Errors with Wrath Classic 3.4.1

### ISSUE THREAD

**Issue by boktai1000** [frozn/TipTac#172]
**Describe the bug**
Lua Error:

```
4x ...libs/LibFroznFunctions-1.0-1/LibFroznFunctions-1.0.lua:1146: attempt to call global 'GetItemLevelColor' (a nil value)
[string "@TipTacTalents/libs/LibFroznFunctions-1.0-1/LibFroznFunctions-1.0.lua"]:1146: in function <...libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:1015>
[string "=(tail call)"]: ?
[string "@TipTacTalents/libs/LibFroznFunctions-1.0-1/LibFroznFunctions-1.0.lua"]:737: in function `InspectDataAvailable'
[string "@TipTacTalents/libs/LibFroznFunctions-1.0-1/LibFroznFunctions-1.0.lua"]:725: in function `?'
[string "@TipTacTalents/libs/LibFroznFunctions-1.0-1/LibFroznFunctions-1.0.lua"]:445: in function <...libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:444>

Locals:
unitID = "Domseven"
callbackForItemData = nil
unitGUID = nil
items = <table> {
 1 = <table> {
 }
 2 = <table> {
 }
 3 = <table> {
 }
 4 = <table> {
 }
 5 = <table> {
 }
 6 = <table> {
 }
 7 = <table> {
 }
 8 = <table> {
 }
 9 = <table> {
 }
 10 = <table> {
 }
 11 = <table> {
 }
 12 = <table> {
 }
 13 = <table> {
 }
 14 = <table> {
 }
 15 = <table> {
 }
 16 = <table> {
 }
 17 = <table> {
 }
 18 = <table> {
 }
 19 = <table> {
 }
}
itemCount = 19
totalScore = 3569
totalItems = 17
totalQuality = 66
totalItemsForQuality = 17
averageItemLevel = 209
totalQualityColor = nil
ignoreInventorySlots = <table> {
 4 = true
 19 = true
}
twoHandedInventoryTypes = <table> {
 17 = true
 15 = true
 26 = true
}
itemMainHand = <table> {
 inventoryType = 13
 effectiveILvl = 213
 quality = 4
}
itemOffHand = <table> {
 inventoryType = 13
 effectiveILvl = 213
 quality = 4
}
isMainHandOnly = false
isMainHandArtifact = false
itemMainHandEffectiveILvl = 213
isOffHandArtifact = false
itemOffHandEffectiveILvl = 213
isSelf = true
avgItemLevel = 209
avgItemLevelEquipped = 209
avgItemLevelPvP = nil
(*temporary) = nil
(*temporary) = "attempt to call global 'GetItemLevelColor' (a nil value)"
LibFroznFunctions = <table> {
 CreatePushArray = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:220
 IsAverageItemLevelAvailable = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:946
 CreateColorFromHexString = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:64
 GetClassColor = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:283
 ChainTables = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:229
 CanInspect = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:593
 GetItemQualityColor = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:292
 CreateMarkupForRoleIcon = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:304
 AreTalentsAvailable = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:801
 GetUnitIDFromGUID = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:358
 CreateMarkupForClassIcon = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:320
 GetAverageItemLevel = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:974
 FormatText = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:92
 CallFunctionAndSuppressErrorMessageAndSpeech = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:259
 IsInspectFrameOpen = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:585
 removeFromTable = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:107
 GameTooltipHookScriptOnTooltipSetUnit = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:338
 InspectUnit = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:479
 GetTalents = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:834
 isWoWFlavor = <table> {
 }
 GetUnitFromTooltip = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:329
 RemoveCallbackFromQueuedInspectCallbacks = <function> defined @TipTacTalents/libs/LibFroznFunctions-1.0/LibFroznFunctions-1.0.lua:519
}

```

**To Reproduce**
Load game with TipTacTalents enabled

**Expected behavior**
No Lua Errors

**Screenshots**

![image](https://user-images.githubusercontent.com/9054563/213309709-3726f9f3-b8f2-40cf-a34d-a73132abe33d.png)

**TipTac Reborn (please complete the following information):**
 - Version 22.12.15

**WoW (please complete the following information):**
 - Flavor: Wrath Classic
 - Version 3.4.1

**Additional context**

**Comment by frozn** [comment]
Strange, this part of the code shouldn't acutally be executed, because the preceeding function `GetAverageItemLevel()` doesn't exist in wotlkc.

Nevertheless I added an existance check for function `GetItemLevelColor()` to prevent this lua error. This change will be included in the next release.

Thanks for reporting this. 👍

**Comment by frozn** [comment]
Fixed with release [v23.01.23](https://github.com/frozn/TipTac/releases/tag/v23.01.23-release).
